packages/shared/charts/src/utils/themeFolderConfig.ts

```diff
@@ -0,0 +1,10 @@
+export async function fetchThemeFolderConfigFile(): Promise<any> {
+    try {
+        const response = await window.fetch(`${window.mx.remoteUrl}com.mendix.charts.json`);
```

Comment:
Maybe we should check `platform` first? seems bit dangerous to use window

Reply:
Also, can you just in few simple steps explain how this file will be added to the project? I mean how this works in general?

Reply:
I don't quite know whether mendix api can provide us a `platform` variable of some kind, but I could do a check on `window` first.

For the steps, check out [the docs](https://docs.mendix.com/refguide/charts-configuration) chapter "4 Chart Theming". Basically add a `com.mendix.charts.json` file with configuration into your mendix project. The only difference with the docs is that it shouldn't go in `[projectRoot]/theme` but in `[projectRoot]/theme/web` now due to atlas changes.